.github/workflows/build-images.yml

```diff
@@ -0,0 +1,70 @@
+name: build-images
+
+on:
+  push:
+    branches:
+      - edge
+      - latest
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build-docker-image:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      # - name: Login to DockerHub
+      #   uses: docker/login-action@v3
+      #   with:
+      #     username: ${{ secrets.DOCKERHUB_USERNAME }}
+      #     password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      # - name: Login to GitHub Container Registry
+      #   uses: docker/login-action@v3
+      #   with:
+      #     registry: ghcr.io
+      #     username: ${{ github.repository_owner }}
+      #     password: ${{ secrets.CR_PAT }}
+
+      - uses: docker/metadata-action@v5
+        id: meta
+        with:
+          images: |
+            docker.io/freshrss/freshrss
+            ghcr.io/${{ github.repository }}
+          tags: |
+            type=edge
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}},enable=${{ !startsWith(github.ref, 'refs/tags/0.') }}
+
+      - name: Build and push
+        id: docker_build_release
+        uses: docker/build-push-action@v5
+        with:
+          file: Docker/Dockerfile
```

Comment:
It would be nice to also build `Docker/Dockerfile-Alpine`

Reply:
Done

Reply:
Just a random question, maybe there is a good answer here I am not seeing: Why multiple prebuilt images? Why not concentrate on one image and test that one instead of multiple ones? The image size difference is not that significant so personally I would concentrate on one image. (But thats personal taste, maybe you have different opinions and thats totally fine)